### PR TITLE
CLDR-17072 Document new unit systems

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-17072

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: docs/ldml/tr35-info.md [hunk 1]
```diff
@@ -895,14 +895,46 @@ The factor and offset can be simple expressions, just like the values in the uni
...
+2. The `number_prefixes`, `dimensionality_prefix`, `si_prefix`, and `binary_prefix` are ignored
+   * Example: systems(square-kilometer) = systems(meter)
+3. Currency units have the `currency` system
+   * Example: systems(curr-usd) = {currency}
+4. Units linked by `-and-`, `-per-`, and *adjacency* are resolved using a modified intersection, where:
+   1. The intersection of {… si …} and {… si_acceptable … } is {… si_acceptable …}
+   2. The intersection of {… metric …} and {… metric_adjacent … } is {… metric_adjacent …}
+Examples: 
+```
+systems(liter-per-hectare) = {si_acceptable metric} ∪ {si_acceptable metric} = {si_acceptable metric}
```
Comment (sffc) [inline docs/ldml/tr35-info.md:932]: ∪ is the union operator; I think you wanted the intersection operator, right?

File: docs/ldml/tr35-info.md [hunk 2]
```diff
@@ -895,14 +895,46 @@ The factor and offset can be simple expressions, just like the values in the uni
...
+
+#### Derived Unit System
+
+The systems attributes also apply to compound units, and are computed in the following way.
+
+1. The `prefixable` system is only applicable to base_components, and is thus removed
+2. The `number_prefixes`, `dimensionality_prefix`, `si_prefix`, and `binary_prefix` are ignored
+   * Example: systems(square-kilometer) = systems(meter)
+3. Currency units have the `currency` system
+   * Example: systems(curr-usd) = {currency}
```
Comment (sffc) [inline docs/ldml/tr35-info.md:926]: What is `curr-usd`? Is that newly defined somewhere that I missed?

**Review by sffc — APPROVED** [review]
Seems good

File: docs/ldml/tr35-info.md (hunk 1 above)
Comment (macchiati) [reply docs/ldml/tr35-info.md:932]: Yup, thanks!

File: docs/ldml/tr35-info.md (hunk 2 above)
Comment (macchiati) [reply docs/ldml/tr35-info.md:926]: It's been defined for a while. Because it is algorithmic, it doesn't show up in the data files.

**Review by sffc — APPROVED** [review]

**Review by pedberg-icu — APPROVED** [review]

**Pull request merged**